websites/I/Incels.is/presence.ts

```diff
@@ -0,0 +1,155 @@
+const presence = new Presence({
+		clientId: "1094962047906758796",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://incels.is/logo/512x512.png",
+}
+
+const tags: string[] = [
+	"Blackpill",
+	"LifeFuel",
+	"SuicideFuel",
+	"RageFuel",
+	"Serious",
+	"Experiment",
+	"JFL",
+	"Venting",
+	"NSFW",
+	"News",
+	"Hypocricy",
+	"LDAR",
+	"Story",
+	"It's Over",
+	"TeeHee",
+	"Soy",
+	"Cope",
+	"Discussion",
+	"Toxic Femininity",
+	"[Whitepill]",
+	"Theory",
+	"Brutal",
+	"Based",
+];
```

Comment:
Why not just get the tag with a queryselector when ur on a thread
```ts
document.querySelector('.label.label--custom-blackpill')?.textContent
```
![image](https://github.com/PreMiD/Presences/assets/42322979/a3e59bc0-3ee3-4c7f-8bb7-46c6c917f26b)

Reply:
The things is, their HTML structure for thread-titles is pretty weird. They have a `<h1>` tag, that contains the title itself, but within said heading is another tag, a `<span>` tag, that contains the thread-tag. So if I do `document.querySelector(h1).textContent`, I not only get the title itself, but also the thread-tag `(eg. "Blackpill This is an example title")`.

What I'm trying to say is, there is no need to get the thread-tags separately, as they come in bundled with the title itself, due to their weird HTML structure. The reason I have this array of tags in my code is so I can take the "title with the tag"-string, find the tag by `.startsWith()` , and put square brackets around it.

Reply:
Why don't you use this then?
```ts
    document.querySelector('[class*="label label--custom-"]')
```
![image](https://github.com/PreMiD/Presences/assets/42322979/a4b32917-513e-43ae-b49b-99ab437af263)

Reply:
With your suggestion, if they decide to add or remove tags, the code will still work as expected. If you had worded your reasoning like that, I would have been with you 100% lol. 

The way you worded your reasoning made it sound like: "Do the same thing you're already doing, but do it slightly differently" lmao.